### PR TITLE
Add meta server field in connection config

### PR DESCRIPTION
**Pull request by cryptoquick**
- [x] CHANGELOG.md updated
  - [x] Rebased/mergable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

Connect #825 #826 

### The problem
We need to send a meta service connection string to the Chronograf backend.

### The Solution
Update the React frontend to support the additional field and request.

### CODE REVIEW

File: ui/src/sources/components/SourceForm.js [hunk 1]
```diff
@@ -96,6 +97,23 @@ export const SourceForm = React.createClass({
...
+                            <a role="button" data-toggle="collapse" className="collapsed" href="#meta-service" style={{fontSize: '14px', fontWeight: 600}}>
+                              <span className="caret"></span>Add Meta Service
+                            </a>
+                          </h2>
+                        </div>
+                        <div className="collapse" id="meta-service" style={{height: '0px'}}>
+                          <div className="panel-body">
+                            <div className="form-group col-xs-12 col-sm-6">
+                              <label htmlFor="meta-url">Connection String</label>
+                              <input type="text" name="metaUrl" ref={(r) => this.metaUrl = r} className="form-control" id="meta-url" placeholder="http://localhost:8086" onChange={onInputChange} value={source.metaUrl || ''}></input>
```
Comment (goller) [inline ui/src/sources/components/SourceForm.js]: placeholder text should be http://localhost:8091

Comment (cryptoquick) [inline ui/src/sources/components/SourceForm.js]: good to know!

**Comment by cryptoquick** [comment]
Hold up on merging this, I'm going to get @alexpaxton 's advice on the styles.

**Comment by cryptoquick** [comment]
Just to be clear, this branch is ready now. It just needs to have the changelog conflict to be resolved.

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,15 +16,41 @@
...
+    - SMTP
+    - Alerta
+
+### UI Improvements
+  1. [#822](https://github.com/influxdata/chronograf/issues/822): Simplify and improve the layout of the Data Explorer
+    - The Data Explorer's intention and purpose has always been the ad hoc and ephemeral exploration of your schema and data.
+      The concept of `Exploration` sessions and `Panels` betrayed this initial intention. The DE turned into a "poor man's"
+      dashboarding tool. In turn, this introduced complexity in the code and the UI. In the future if I want to save, manipulate,
+      and view multiple visualizations this will be done more efficiently and effectively in our dashboarding solution.
+>>>>>>> master
```
Comment (cryptoquick) [inline CHANGELOG.md]: >:(

**Pull request merged**